chart/templates/db-migrate-job.yaml

```diff
@@ -19,6 +19,12 @@ spec:
           command: ["bundle"]
           args:  ["exec", "rails", "db:migrate"]
           volumeMounts:
+            - name: vets-api-mockdata
```

Comment:
I don't think db migrate needs mockdata or clamav volumes

Reply:
removed - they lines were only there for consistency with existing